elasticjob-infra/elasticjob-infra-common/src/main/java/org/apache/shardingsphere/elasticjob/infra/handler/sharding/impl/RoundRobinByNameJobShardingStrategy.java

```diff
@@ -38,7 +38,13 @@ public Map<JobInstance, List<Integer>> sharding(final List<JobInstance> jobInsta
     
     private List<JobInstance> rotateServerList(final List<JobInstance> shardingUnits, final String jobName) {
         int shardingUnitsSize = shardingUnits.size();
-        int offset = Math.abs(jobName.hashCode()) % shardingUnitsSize;
+        int jobHashCode = jobName.hashCode();
+        int offset = 0;
+        if (jobHashCode != Integer.MIN_VALUE) {
+            offset = Math.abs(jobHashCode) % shardingUnitsSize;
+        } else {
+            offset = Integer.MIN_VALUE % shardingUnitsSize;
+        }
```

Comment:
Hi @zume0127 
Could you complete the tests `org.apache.shardingsphere.elasticjob.infra.handler.sharding.impl.RotateServerByNameJobShardingStrategyTest` for your logic?